.env.example

```
@@ -1,42 +1,42 @@
 # SkyStream Production Environment Configuration
 
 # Application Configuration
-VITE_APP_NAME=SkyStream
-VITE_APP_VERSION=2.0.0
-VITE_APP_DESCRIPTION=Your ultimate destination for streaming movies and TV shows
+NEXT_PUBLIC_APP_NAME=SkyStream
+NEXT_PUBLIC_APP_VERSION=2.0.0
+NEXT_PUBLIC_APP_DESCRIPTION=Your ultimate destination for streaming movies and TV shows
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_APP_DESCRIPTION key should go before the NEXT_PUBLIC_APP_NAME key

(UnorderedKey)